builtin/providers/aws/json.go

```diff
@@ -0,0 +1,38 @@
+package aws
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func normalizeJson(jsonString interface{}) string {
+	if jsonString == nil || jsonString == "" {
+		return ""
+	}
+	var j interface{}
+	err := json.Unmarshal([]byte(jsonString.(string)), &j)
+	if err != nil {
+		return fmt.Sprintf("Error parsing JSON: %s", err)
+	}
+	b, err := json.Marshal(j)
+	if err != nil {
+		panic(err) // unexpected
+	}
+	return string(b)
+}
+
+func normalizePolicyDocument(policyString interface{}) string {
```

Comment:
> If in the future you make a change to IAMPolicyDoc, you almost certainly do not need to make a change to `normalizePolicyDocument`

Agreed.

> If in the future you make a change to `normalizeJson`, you almost certainly also want to make a change to `normalizePolicyDocument`.

Why do you think so? `normalizeJson` is a generic function unrelated to IAM, it can be used to normalize CloudFormation templates for example (JSON).